packages/dd-trace/src/debugger/devtools_client/index.js

```diff
@@ -179,32 +214,40 @@ session.on('Debugger.paused', async ({ params }) => {
       }
     }
 
+    let message = ''
+    if (probe.templateRequiresEvaluation) {
+      const results = evalResults[messageIndex++]
+      if (results === undefined) {
+        log.error('[debugger:devtools_client] No evaluation results for probe %s', probe.id)
+      } else {
+        for (const result of results) {
+          if (typeof result === 'string') {
+            message += result
+          } else {
+            // If `result` isn't a string, it's an evaluation error object
+            if (snapshot.evaluationErrors === undefined) {
+              snapshot.evaluationErrors = [result]
+            } else {
+              snapshot.evaluationErrors.push(result)
+            }
+            message += `{${result.message}}`
```

Comment:
Should that maybe use the stack information instead?

Reply:
The stack doesn't provide any meaningful information to the end user (it's just internal frames). Showing the error message only in curly brases is the standard we have used across all tracing languages, so here I'm just aligning with the rest of the tracers.